code/modules/research/designs/mecha_designs.dm

```diff
@@ -373,4 +373,48 @@
 	build_path = /obj/item/mecha_parts/mecha_equipment/weapon/energy/plasma
 	materials = list(MAT_METAL = 1500, MAT_GLASS = 500, MAT_PLASMA = 200)
 	construction_time = 100
-	category = list("Exosuit Equipment")
\ No newline at end of file
+	category = list("Exosuit Equipment")
+
+/datum/design/mech_taser
+	name = "Exosuit Weapon (PBT \"Pacifier\" Mounted Taser)"
+	desc = "A weapon for combat exosuits. Shoots non-lethal stunning electrodes."
+	id = "mech_taser"
+	build_type = MECHFAB
+	req_tech = list("combat" = 2)
+	build_path = /obj/item/mecha_parts/mecha_equipment/weapon/energy/taser
+	materials = list(MAT_METAL=10000)
+	construction_time = 100
+	category = list("Exosuit Equipment")
+
+/datum/design/mech_lmg
+	name = "Exosuit Weapon (\"Ultra AC 2\" LMG)"
+	desc = "A weapon for combat exosuits. Shoots a rapid, three shot burst."
+	id = "mech_lmg"
+	build_type = MECHFAB
+	req_tech = list("combat" = 2)
+	build_path = /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/lmg
+	materials = list(MAT_METAL=10000)
+	construction_time = 100
+	category = list("Exosuit Equipment")
+
+/datum/design/mech_sleeper
+	name = "Exosuit Medical Equipment (Mounted Sleeper)"
+	desc = "Equipment for medical exosuits. A mounted sleeper that stabilizes patients and can inject reagents in the exosuit's reserves."
+	id = "mech_sleeper"
+	build_type = MECHFAB
+	req_tech = list("biotech" = 2)
+	build_path = /obj/item/mecha_parts/mecha_equipment/sleeper
+	materials = list(MAT_METAL=5000,MAT_GLASS=10000)
+	construction_time = 100
+	category = list("Exosuit Equipment")
+
+/datum/design/mech_syringe_gun
+	name = "Exosuit Medical Equipment (Syringe Gun)"
+	desc = "Equipment for medical exosuits. A chem synthesizer with syringe gun. Reagents inside are held in stasis, so no reactions will occur."
+	id = "mech_syringe_gun"
+	build_type = MECHFAB
+	req_tech = list("magnets" = 3,"biotech" = 3)
+	build_path = /obj/item/mecha_parts/mecha_equipment/syringe_gun
+	materials = list(MAT_METAL=3000,MAT_GLASS=2000)
+	construction_time = 200
+ 	category = list("Exosuit Equipment")
```

Comment:
We spent a while trying to figure this out on coderbus, but it turns out you left a stray space in the beginning of this line. Delete it (leaving only the tab) and it should compile fine.